tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -611,8 +611,17 @@ private static ArgumentParser argParser() {
                 .action(store())
                 .required(false)
                 .type(String.class)
-                .metavar("CONFIG_FILE")
-                .help("Consumer config properties file (config options shared with command line parameters will be overridden).");
+                .metavar("CONFIG-FILE")
+                .help("(DEPRECATED) Consumer config properties file (config options shared with command line parameters will be overridden)." +
+                        "This option will be removed in a future version. Use --command-config instead.");
+
+        parser.addArgument("--command-config")
+                .action(store())
+                .required(false)
+                .type(String.class)
+                .metavar("CONFIG-FILE")
+                .dest("commandConfigFile")
+                .help("Config properties file.");
```

Comment:
I suggest adding `(config options shared with command line parameters will be overridden)` to the help string. That's still true.